### ISSUE TITLE
Does SnapPlus have any special code to work via sideloading?

### ISSUE THREAD

**Issue by ghost** [aviralapurv/Snapchat-Hack-Tweakpatcher#1]
I was just curious if SnapPlus has any special code in it to work via sideloading and if it doesnt, could you make a tutorial on how we could replace the already .sh file with another tweak instead of SnapPlus

**Comment by aviralapurv** [comment]
@seiterseiter yes snapPlus does have some codes to enhance its feature ,more than the stock snapchat app. .. i'll try another tutorial as well .. i'll work on it and let you know;)
